fgcm/fgcmStars.py

```diff
@@ -2129,9 +2129,11 @@ def plotRefStarColorTermResiduals(self, fgcmPars):
 
                 ax.hexbin(gmiGRS[refUse], delta, bins='log', extent=[xlow, xhigh, ylow, yhigh])
 
-                binstruct = dataBinner(gmiGRS[refUse], delta, 0.1, [xlow, xhigh], nTrial=10)
-                ok, = np.where(binstruct['N'] > 0)
-                ax.plot(binstruct['X'][ok], binstruct['Y'][ok], 'r--')
+                # Only do the binning if we have data
+                if xhigh > xlow and refUse.size > 10:
+                    binstruct = dataBinner(gmiGRS[refUse], delta, 0.1, [xlow, xhigh], nTrial=10)
+                    ok, = np.where(binstruct['N'] > 0)
+                    ax.plot(binstruct['X'][ok], binstruct['Y'][ok], 'r--')
```

Comment:
Do you not need to do this check for the `hexbin` plot above?

Reply:
Oh dear, not there but above if there are nothing at all, you're right.